src/components/Comments.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router";
+import { getComments } from "../api";
+
+const Comments = () => {
+  const { review_id } = useParams();
+  const [comments, setComments] = useState(undefined);
+  const [commentLoad, setCommentLoad] = useState(true);
```

Comment:
Good use of the isLoading to indicate to the user if we're still waiting for content back